pkg/registry/service/registry.go

```diff
@@ -81,7 +85,19 @@ func (s *storage) UpdateService(ctx api.Context, svc *api.Service) (*api.Service
 }
 
 func (s *storage) WatchServices(ctx api.Context, options *api.ListOptions) (watch.Interface, error) {
-	return s.Watch(ctx, options)
+	return s.ss.Watch(ctx, options)
+}
+
+func (s *storage) ExportService(ctx api.Context, name string, options unversioned.ExportOptions) (*api.Service, error) {
+	exporter, isExporter := s.ss.(rest.Exporter)
```

Comment:
If you added the name to get this line to work, the way to specify it without the name change is `s.StandardStorage.(rest.Exporter)`